dask/array/core.py

```diff
@@ -782,6 +784,9 @@ def finalize(results):
 class Array(Base):
     """ Parallel Array
 
+    Do not construct Dask.arrays with this class.  Instead use
+    ``da.from_array``.
```

Comment:
Rather than "Not user facing" I would say "For advanced uses." Some users may be more sophisticated than others, and my understanding is that this *is* public API :).